src/VisualStudioExtension/src/Tanzu.Toolkit.VisualStudioExtension.csproj

```diff
@@ -167,6 +167,9 @@
     <PackageReference Include="Microsoft.Extensions.Logging.Abstractions">
       <Version>6.0.0</Version>
     </PackageReference>
+    <PackageReference Include="System.Collections.Immutable">
+      <Version>5.0.0</Version>
+    </PackageReference>
```

Comment:
What's this for? I missed it

Reply:
`Should also clear build warning for Systems.Collections.Immutable`